src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -94,8 +107,18 @@
  *   &lt;property name="maximum" value="10"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
```

Comment:
Please review configs, and make sure all properties of Checks are showed in examples. You are ok to create new configs/samples for them.

One of property is file extension. So for config you should have two code blocks with definition of name outside of pre tag

Reply:
@romani  made the changes. could you review them